Automattic/wp-calypso
# Goals Capture: Record calypso_signup_intent_select with import prop.

#### Proposed Changes

* Also record the `calypso_signup_intent_select` event with import prop.
* This is to ensure funnels using this tracks event continue to work, while `import` prop adds backward compatibility to the `import` intent which is now deprecated (absorbed by other intents).

Note: This is currently branched out from https://github.com/Automattic/wp-calypso/pull/64724. Once the parent PR is merged, this can be rebased.

#### Testing Instructions

1. Go to the Goals Step via `/setup/goals?siteSlug=your_site_here.wordpress.com&flags=signup/goals-step`.
2. Run `localStorage.setItem( 'debug', 'calypso:analytics*' );` in the devtool console.
3. Select one goal, multiple goals, no goals and click on the Continue button.
   - Also include the `import` goal occasionally.
5. From the devtool console, check if the `calypso_signup_intent_select` event is fired and the prop values are correct.
